cardano-testnet/src/Testnet/Components/SPO.hs

```diff
@@ -401,3 +405,35 @@ registerSingleSpo identifier tap@(TmpAbsolutePath tempAbsPath') nodeConfigFile s
               poolColdVkeyFp
               currentRegistedPoolsJson
   return (poolId, poolColdSkeyFp, poolColdVkeyFp, vrfSkeyFp, vrfVkeyFp)
+
+-- | Generates Stake Pool Operator (SPO) voting files, using @cardano-cli@.
+--
+-- Returns a list of generated @File VoteFile In@ representing the paths to
+-- the generated voting files.
+generateVoteFiles :: (MonadTest m, MonadIO m, MonadCatch m)
```

Comment:
```suggestion
generateVoteFiles :: (HasCallStack, MonadTest m, MonadIO m, MonadCatch m)
```

Reply:
Fixed in: https://github.com/IntersectMBO/cardano-node/pull/5809/commits/92abf97e8111a5d0ae3dade7f1721d5e3da5a2e7